collectors/profile.plugin/README.md

```diff
@@ -0,0 +1,5 @@
+You can configure this plugin with the following options:
+    - `profplug charts = 500`: number of charts to create
```

Comment:
If somehow this configuration is inside a `/etc/netdata/profile.d.conf`, I suggest to remove the `profplug` from the names.

On the other hand, if this is inside `/etc/netdata/netdata.conf`,  I suggest to move for an external configuration file.